docs/Developers/Debug_Bears.rst

```diff
@@ -122,3 +122,88 @@ Or to debug all bears specified by ``bears`` setting:
     bears = PEP8Bear,MypyBear
     files = <filename>
     debug_bears = True
+
+Debugger features
```

Comment:
Use title case for headings -> `Debugger Features`
Same for all other titles below (like `Inspect Bear Settings`). You can read about title case [here](https://www.grammar-monster.com/lessons/capital_letters_title_case.htm)